src/Apps/Home/Components/HomeContentCards.tsx

```diff
@@ -5,6 +5,70 @@ import {
   StaticHeroUnit,
 } from "Apps/Home/Components/HomeHeroUnits/HomeHeroUnit"
 import { CaptionedImage as BrazeContentCard } from "@braze/web-sdk"
+import {
+  Box,
+  ResponsiveBox,
+  GridColumns,
+  Column,
+  SkeletonBox,
+  SkeletonText,
+  Spacer,
+} from "@artsy/palette"
+import { Media } from "Utils/Responsive"
+
+const FakeCard = () => {
+  return (
+    <GridColumns bg="black5" width="100%">
```

Comment:
Why is the whole thing this grey?

Reply:
That's just copy/paste from the herounit component! What do you think it should be? 🤔

Reply:
Oh, you're right. Somehow missed that 👍